cmake_modules/commonSetup.cmake

```diff
@@ -676,26 +676,40 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
         message(STATUS "Using externals directory at ${EXTERNALS_DIRECTORY}")
     endif()
 
-  IF ( NOT MAKE_DOCS_ONLY )
-      IF ("${EXTERNALS_DIRECTORY}" STREQUAL "")
+    IF ( NOT MAKE_DOCS_ONLY )
+      FIND_PACKAGE(BISON)
+      FIND_PACKAGE(FLEX)
+      IF ( BISON_FOUND AND FLEX_FOUND )
+        SET(BISON_exename ${BISON_EXECUTABLE})
+        SET(FLEX_exename ${FLEX_EXECUTABLE})
         IF (WIN32)
-          SET(bisoncmd "win_bison")
-          SET(flexcmd "win_flex")
+          SET(bisoncmd ${BISON_exename})
+          SET(flexcmd ${FLEX_exename})
         ELSE()
           SET(bisoncmd "bison")
           SET(flexcmd "flex")
         ENDIF()
-      ELSE()
-        IF (WIN32)
-          SET(bisoncmdprefix "call")
-          SET(flexcmdprefix "call")
-          SET(bisoncmd "${EXTERNALS_DIRECTORY}\\bison\\bison.bat")
-          SET(flexcmd "${EXTERNALS_DIRECTORY}\\bison\\flex.bat")
-        ELSE ()
-          SET(bisoncmd "${EXTERNALS_DIRECTORY}/bison/bison")
-          SET(flexcmd "${EXTERNALS_DIRECTORY}/bison/flex")
+      ELSE ( BISON_FOUND AND FLEX_FOUND )
```

Comment:
trivial: Not sure I like the condition repeated here - I assumed it was an ELSEIF, but don't really care

Reply:
For consistency I shall remove.